Fail
file path is :
projects/incubator-milagro-crypto/rust/src/nhs.rs

output is:


error is :
   Compiling amcl v0.3.0 (/home/k/ogs_PHD/LLM4SE/2Rust_projects/auto_get_functionPair/projects/incubator-milagro-crypto/rust)
error[E0425]: cannot find function, tuple struct or tuple variant `Error` in this scope
   --> src/nhs.rs:536:5
    |
457 | fn error(rng: &mut RAND, poly: &mut [i32]) {
    | ------------------------------------------ similarly named function `error` defined here
...
536 |     Error(rng, &mut e);
    |     ^^^^^
    |
help: a function with a similar name exists
    |
536 |     error(rng, &mut e);
    |     ~~~~~
help: consider importing one of these items
    |
22  + use core::fmt::Error;
    |
22  + use std::fmt::Error;
    |

error[E0425]: cannot find function, tuple struct or tuple variant `Error` in this scope
   --> src/nhs.rs:537:5
    |
457 | fn error(rng: &mut RAND, poly: &mut [i32]) {
    | ------------------------------------------ similarly named function `error` defined here
...
537 |     Error(rng, &mut s);
    |     ^^^^^
    |
help: a function with a similar name exists
    |
537 |     error(rng, &mut s);
    |     ~~~~~
help: consider importing one of these items
    |
22  + use core::fmt::Error;
    |
22  + use std::fmt::Error;
    |

error[E0425]: cannot find function, tuple struct or tuple variant `Error` in this scope
   --> src/nhs.rs:536:5
    |
457 | fn error(rng: &mut RAND, poly: &mut [i32]) {
    | ------------------------------------------ similarly named function `error` defined here
...
536 |     Error(rng, &mut e);
    |     ^^^^^
    |
help: a function with a similar name exists
    |
536 |     error(rng, &mut e);
    |     ~~~~~
help: consider importing one of these items
    |
22  + use core::fmt::Error;
    |
22  + use serde::export::fmt::Error;
    |
22  + use serde::private::ser::Error;
    |
22  + use std::fmt::Error;
    |

error[E0425]: cannot find function, tuple struct or tuple variant `Error` in this scope
   --> src/nhs.rs:537:5
    |
457 | fn error(rng: &mut RAND, poly: &mut [i32]) {
    | ------------------------------------------ similarly named function `error` defined here
...
537 |     Error(rng, &mut s);
    |     ^^^^^
    |
help: a function with a similar name exists
    |
537 |     error(rng, &mut s);
    |     ~~~~~
help: consider importing one of these items
    |
22  + use core::fmt::Error;
    |
22  + use serde::export::fmt::Error;
    |
22  + use serde::private::ser::Error;
    |
22  + use std::fmt::Error;
    |

error[E0599]: no method named `get_byte` found for mutable reference `&mut RAND` in the current scope
   --> src/nhs.rs:531:23
    |
531 |         seed[i] = rng.get_byte() as u8;
    |                       ^^^^^^^^ help: there is a method with a similar name: `getbyte`

error[E0425]: cannot find function, tuple struct or tuple variant `Parse` in this scope
   --> src/nhs.rs:534:5
    |
352 | fn parse(seed: &[u8], poly: &mut [i32]) {
    | --------------------------------------- similarly named function `parse` defined here
...
534 |     Parse(&seed, &mut b);
    |     ^^^^^ help: a function with a similar name exists: `parse`

error[E0599]: no method named `get_byte` found for mutable reference `&mut rand::RAND` in the current scope
   --> src/nhs.rs:531:23
    |
531 |         seed[i] = rng.get_byte() as u8;
    |                       ^^^^^^^^ help: there is a method with a similar name: `getbyte`

Some errors have detailed explanations: E0425, E0599.
For more information about an error, try `rustc --explain E0425`.
error: could not compile `amcl` (lib) due to 4 previous errors
warning: build failed, waiting for other jobs to finish...
error: could not compile `amcl` (lib test) due to 4 previous errors

